### DISCUSSION TITLE
RunTimeError: Unfulfilled dependency at run time ...

### DISCUSSION THREAD

**Discussion by pamelaxu213** [broadinstitute/seqr#3402] [Q&A]
Hello,

I am encountering this error message while uploading VCFs to seqr:
`Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/luigi/worker.py", line 184, in run
    raise RuntimeError('Unfulfilled %s at run time: %s' % (deps, ', '.join(missing)))
RuntimeError: Unfulfilled dependency at run time: VcfFile__input_vcfs_GRCh_c42c6fecdd
DEBUG: 1 running tasks, waiting for next task to finish
INFO: Informed scheduler that task   SeqrVCFToMTTask__seqr_reference__VARIANTS__input_vcfs_GRCh_b92498a6b1   has status   FAILED`

Could you help me understand what the error message entails?  I have three WGS VCFs, 2 of which have this error

**Comment by hanars** [comment]
That sounds like you did not specify an input VCF, or that the input VCF you specified does not exits

**Reply by pamelaxu213** [reply]
It was indeed, my file path was wrong.

Thank you!
